packages/cactus-plugin-keychain-aws-sm/README.md

```diff
@@ -1,3 +1,164 @@
 # `@hyperledger/cactus-plugin-keychain-aws-sm`
 
-## TO-DO
\ No newline at end of file
+- [`@hyperledger/cactus-plugin-keychain-aws-sm`](#hyperledgercactus-plugin-keychain-aws-sm)
+  - [1. Usage](#1-usage)
+    - [1.1. Installation](#11-installation)
+    - [1.2. Using as a Library](#12-using-as-a-library)
+    - [1.3. Using via the API Client](#13-using-via-the-api-client)
+  - [2. Architecture](#2-architecture)
+    - [2.1. set-keychain-entry-endpoint](#21-set-keychain-entry-endpoint)
+    - [2.2. get-keychain-entry-endpoint](#22-get-keychain-entry-endpoint)
+    - [2.3. has-keychain-entry-endpoint](#23-has-keychain-entry-endpoint)
+    - [2.4. delete-keychain-entry-endpoint](#24-delete-keychain-entry-endpoint)
+  - [3. Prometheus Exporter](#3-prometheus-exporter)
```

Comment:
@jagpreetsinghsasan Please put the Promehetus related sections one level lower, e.g. there should be a top level `Monitoring` section and then everything about the prometheus monitoring should be under that (and later if we have other monitoring topics to discuss they can fit in there)